node/hubController.js

```diff
@@ -289,7 +287,8 @@ class HubController {
     }
 
     _invokeMethod(translator, deviceInfo, methodName, params) {
-        console.log("----------------- _invokeMethod " + methodName + " with params "+ JSON.stringify(params));
+        this.ConsoleLogger.verbose(
+            `_invokeMethod ${methodName} with params ${JSON.stringify(params)}`);
```

Comment:
yes

---
In reply to: [103834273](https://github.com/openT2T/cloud/pull/26#discussion_r103834273) [](ancestors = 103834273,103828178)